src/scss/partials/_logo.scss

```diff
@@ -41,8 +41,8 @@ $logo-color--transparent    : $white;
     margin-left: -10.5px; // half of hamburger menu width
 
     @include media('>=mid') {
-        width: 163px;
-        height: 40px;
+        width: 130px;
```

Comment:
Have you got a link to the designs on zeplin to check these sizes against? Want to make sure they aren't orderweb specific as the previous sizes came from the header designs design gave us.